airflow/ui/src/pages/DagsList.tsx

```diff
@@ -41,9 +41,9 @@ import { FiSearch } from "react-icons/fi";
 
 import { DAG } from "openapi/requests/types.gen";
 import { useDagServiceGetDags } from "openapi/queries";
-import { DataTable } from "./components/DataTable";
-import { pluralize } from "./utils/pluralize";
-import { useTableURLState } from "./components/DataTable/useTableUrlState";
+import { DataTable } from "../components/DataTable";
```

Comment:
Is it a good idea to setup absolute imports for `src` like `@/components/DataTable`?

Reply:
Yes, I think we will update import paths after https://github.com/apache/airflow/pull/42184